### PR TITLE
choice(larray)

### PR DESCRIPTION
**Pull request by gdementen**
a small PR on top of #644

### CODE REVIEW

**Comment by gdementen** [comment]
oops, I forgot to document the extra functionality in choices docstring

**Comment by gdementen** [comment]
I wonder if choice(p=larray), wouldn't be more logical than choice(choices=larray) 
That is getting choices from the probabilities array, instead of taking the probabilities from the choices array.

File: larray/random.py [hunk 1]
```diff
@@ -0,0 +1,456 @@
...
+        If an LArray, a random sample is generated using its axis as choices and values as probabilities.
+        If an ndarray, a random sample is generated from its elements.
+        If an int n, the random sample is generated as if choices was la.sequence(n)
+    axes : int, tuple of int, str, Axis or tuple/list/AxisCollection of Axis, optional
+        Axes (or shape) of the resulting array. If ``axes`` is None (the default), a single value is returned.
+        Otherwise, if the resulting axes have a shape of, e.g., ``(m, n, k)``, then ``m * n * k`` samples are drawn.
+    replace : boolean, optional
+        Whether the sample is with or without replacement.
+    p : 1-D array-like, optional
+        The probabilities associated with each entry in a.
```
Comment (alixdamman) [inline larray/random.py]: s/a/choices/ ?

File: larray/random.py [hunk 2]
```diff
@@ -0,0 +1,456 @@
...
+    --------
+    LArray or scalar
+        The generated random samples with given ``axes`` (or shape).
+
+    Raises
+    -------
+    ValueError
+        If a is an int and less than zero, if a or p are not 1-dimensional, if a is an array-like of size 0,
+        if p is not a vector of probabilities, if a and p have different lengths, or
+        if replace=False and the sample size is greater than the population size.
```
Comment (alixdamman) [inline larray/random.py]: s/a/choices/ in all `Raises` section

File: larray/random.py (hunk 1 above)
Comment (gdementen) [reply larray/random.py]: good catch. Bad copy-paste from numpy.

File: larray/random.py (hunk 2 above)
Comment (gdementen) [reply larray/random.py]: idem

**Comment by gdementen** [comment]
@alixdamman : your opinion on this:
> I wonder if choice(p=larray), wouldn't be more logical than choice(choices=larray)
That is getting choices from the probabilities array, instead of taking the probabilities from the choices array.

?

**Comment by alixdamman** [comment]
It makes sense to me.

**Pull request merged**